power/property_contexts

```diff
@@ -1,2 +1,4 @@
 # property for power HAL
 vendor.power_hal.          u:object_r:vendor_power_hal_prop:s0
+
+ro.power.fixed_performance_scale_factor u:object_r:exported_default_prop:s0 exact int
```

Comment:
the name of a read only property defined by a vendor should be prefixed with ro.vendor, or it may not pass the VTS test. And assign "exported_default_prop" to a vendor defined proeprty may also violated the VTS requirement, the type should be prefixed with "vendor_"

Reply:
Will be raising a separate patch for this System property definition